__tests__/unit/graphql/loaders/createLoaderWithSpecificKey.test.js

```diff
@@ -43,33 +43,20 @@ describe("createLoaderWithSpecificKey: dataloader with specific parameter for gr
         mockingoose(Competition).toReturn(competition_data, "find");
     });
     test("Should return array with the correct number of elements.", async () => {
-        const result = await loader.loadMany(["609aa4bde6483525a06b8e5b"]);
-        expect(result).toHaveLength(1);
+        const result = await loader.loadMany(["609aa4bde6483525a06b8e5b","609aa4bde6483525a06b8e5z"]);
+        expect(result).toHaveLength(2);
     });
```

Comment:
Statements like "correct number of elements" are meaningless.
Try "should return array with object for every requested id" or something similar.